### PR TITLE
Set timestamp of layer files to 1 instead of 0

### PR DESCRIPTION
**Pull request by mbonato**
About #1079.

### CODE REVIEW

**Comment by loosebazooka** [comment]
A little update here: https://github.com/GoogleContainerTools/jib/issues/1079#issuecomment-428983535

@mbonato if you would like to continue this PR, that would be great. If you are unable to or wish not to for any reason, a jib team member will fix this in a new PR.

**Comment by mbonato** [comment]
I signed it!

File: jib-core/src/main/java/com/google/cloud/tools/jib/image/ReproducibleLayerBuilder.java [hunk 1]
```diff
@@ -113,7 +113,7 @@ public UnwrittenLayer build() {
     TarStreamBuilder tarStreamBuilder = new TarStreamBuilder();
     for (TarArchiveEntry entry : sortedFilesystemEntries) {
       // Strips out all non-reproducible elements from tar archive entries.
-      entry.setModTime(0);
+      entry.setModTime(1);
```
Comment (loosebazooka) [inline jib-core/src/main/java/com/google/cloud/tools/jib/image/ReproducibleLayerBuilder.java]: I think we decided to use 1000 instead of 1.

Comment (mbonato) [reply jib-core/src/main/java/com/google/cloud/tools/jib/image/ReproducibleLayerBuilder.java]: I fully agree. I just tried to sign the contributors license agreement. Please feel free to close this PR.

**Comment by mbonato** [comment]
I signed it!

File: jib-core/src/main/java/com/google/cloud/tools/jib/image/ReproducibleLayerBuilder.java (hunk 1 above)
Comment (loosebazooka) [reply jib-core/src/main/java/com/google/cloud/tools/jib/image/ReproducibleLayerBuilder.java]: Okay, will do that. We still appreciate the contribution!

**Comment by loosebazooka** [comment]
@mbonato The commit author needs to match your public github email address for the CLA to register.

Currently author is:  `Martin Bonato <mbonato@Martins-MacBook-Pro.local>`

you can change it by amending the author on the commit:
`git commit --amend --author="Martin Bonato <your.github@email.address>"`

You will need to force push to update your fork with the new author information

**Comment by mbonato** [comment]
@loosebazooka sorry for that. I reinstalled software and forgot to set the proper user.email config.

**Comment by mbonato** [comment]
I've set the modTime to 1000 (1 second).

**Comment by chanseokoh** [comment]
Thanks for your contribution, @mbonato. We just filed [another PR with a test](https://github.com/GoogleContainerTools/jib/pull/1133), so I'll close this.

**Pull request closed without merging**